operator/pkg/onepassword/items.go

```diff
@@ -33,6 +35,30 @@ func GetOnePasswordItemByPath(opConnectClient connect.Client, path string) (*one
 	return item, nil
 }
 
+func ParseReference(reference string) (string, string, error) {
```

Comment:
The error messages are highlighting that the references should be in the form `op://<vault>/<item>/<field>` but we aren't parsing the field. Is that specificity being handled elsewhere?

Reply:
This method also takes the `op://` string and parses out the item and vault values